scripts/typedoc/generate_typedoc.sh

```diff
@@ -6,4 +6,7 @@
 
 # List the commands for each service here:
 npx typedoc --theme default --entryPointStrategy expand --excludeExternals \
-    --tsconfig tsconfig.json --out apidocs auth cloudant --includeVersion
+    --excludeInternal --tsconfig tsconfig.json --out apidocs auth cloudant --includeVersion \
+    --exclude cloudant/features/changesParamsHelper.ts \
+    --exclude cloudant/features/changesResultItemStream.ts \
+    --exclude cloudant/features/changesResultIterator.ts
```

Comment:
`/** @internal */` with `--excludeInternal` hides only the class, but not the file.
Without explicitly excluding e.g. `cloudant/features/changesParamsHelper.ts`, it is going to be reachable as an empty module (`Module cloudant/features/changesParamsHelper`). 

Currently we have 2 links: https://ibm.github.io/cloudant-node-sdk/docs/latest/modules/cloudant_features_changesParamsHelper.html for the "module" and https://ibm.github.io/cloudant-node-sdk/docs/latest/classes/cloudant_features_changesParamsHelper.ChangesParamsHelper.html for the class. I think we do not need any of them in the docs.

I can remove the `/** @internal */` for the classes in the excluded files, it should not matter as the matter of the visibility.

> I can't actually see this file being executed anywhere, am I missing something

Thanks, yeah I did not notice. I guess then this file is no needed at all.

Reply:
If we do not stick to the current (left-side) navigation on the published page we can make `index.ts` as an entrypoint instead of `cloudant` and `auth`. That would expose classess and namespaces we expose in the code too. With that we can eliminate the `--exclude`s and some of the `/** @internal */` comments. I leave here the output screenshot of the status when these configuration options:
```json
{
    "theme": "default",
    "entryPointStrategy": "expand",
    "excludeExternals": true,
    "tsconfig": "tsconfig.json",
    "out": "apidocs",
    "includeVersion": true,
    "entryPoints": ["index.ts"],
    "excludeInternal": true
}
```
![image](https://github.com/IBM/cloudant-node-sdk/assets/17944122/61c738fd-3497-454b-bd02-b590a0416619)

Reply:
Okay, I did this in 3 commits:
[5a9849f](https://github.com/IBM/cloudant-node-sdk/pull/1434/commits/5a9849fb6549ff2357f438b15d4cddba28a5c91a)
[827deed](https://github.com/IBM/cloudant-node-sdk/pull/1434/commits/827deed2e724925e9dafb913de5ba61873be0453)
[8995641](https://github.com/IBM/cloudant-node-sdk/pull/1434/commits/8995641fbdd2b77c432bd77e50576baa0ddac513)